external/Makefile.tools

```diff
@@ -11,7 +12,6 @@ spike_dir       := $(BP_EXTERNAL_DIR)/riscv-isa-sim
 dromajo_dir     := $(BP_EXTERNAL_DIR)/dromajo
 axe_dir         := $(BP_EXTERNAL_DIR)/axe
 dramsim2_dir    := $(BP_EXTERNAL_DIR)/DRAMSim2
```

Comment:
probably want to get rid of dramsim2 here too?

Reply:
Yeah. Also, we can remove both DRAMSim2 and DRAMSim3 submodules.